tasks/main.yml

```
@@ -60,7 +60,7 @@
 
 - name: Include services management
   import_tasks: services.yml
-  when: consul_services is defined and consul_services|length>0
+  when: consul_services is defined and consul_services | length>0
```

```suggestion
  when: consul_services is defined and consul_services | length > 0
```

I wish `ansible-lint` detected this, but it doesn't.